style.css

```diff
@@ -420,3 +420,103 @@ ul {
   color: #e2505c;
   border: 1px solid #e2505c;
 }
+
+.form-section {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  grid-gap: 24px;
+  background-image: url(./images/image1.png);
+  background-repeat: no-repeat;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 288px;
+}
+
+form > input {
+  height: 52px;
+  padding-top: 4px;
+  padding-bottom: 33px;
+  padding-left: 16px;
+  border-radius: 8px;
```

Comment:
- [ ] Please make sure that your project is responsive on a mobile screen. but as you can see from the screenshot below, you have all sections of your project not responsive at a screen width of `375px` :smile: Please kindly fix this by adjusting the style of your sections.
- [ ] After fixing this, your sections should cover the mobile screen and there should be no white space left on the right side

![Screenshot from 2022-12-07 09-54-40](https://user-images.githubusercontent.com/92303891/206149432-9fa856be-7ce4-4e3f-bf48-f4a751a683bd.png)